RELEASE_NOTES.md

```diff
@@ -1,7 +1,8 @@
 # Release notes
 
-## New in git-machete 3.20.1
+## New in git-machete 3.21.0
 
+- add: `--urls` flag to `git machete github anno-prs` that also adds the URL of the PR to the annotation (contributed by @guyboltonking)
```

Comment:
Also, config key to be added that would affect all subcommands of `github`

Reply:
As for now, I'd rather leave the flag just in `anno-prs` to keep CLI bloat at bay